score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/ComputeServiceImpl.java

```diff
@@ -1,21 +1,24 @@
 package io.cloudslang.content.jclouds.services.impl;
 
+import io.cloudslang.content.jclouds.entities.constants.Constants;
+import io.cloudslang.content.jclouds.entities.inputs.CommonInputs;
+import io.cloudslang.content.jclouds.entities.inputs.CustomInputs;
 import io.cloudslang.content.jclouds.services.ComputeService;
-import io.cloudslang.content.jclouds.services.JcloudsComputeService;
+import io.cloudslang.content.jclouds.services.JCloudsComputeService;
 import org.jclouds.ContextBuilder;
 import org.jclouds.compute.ComputeServiceContext;
 import org.jclouds.compute.domain.ComputeMetadata;
 import org.jclouds.domain.Location;
+import org.jclouds.ec2.domain.Reservation;
+import org.jclouds.ec2.domain.RunningInstance;
 
 import java.util.HashSet;
 import java.util.Set;
 
 /**
  * Created by persdana on 6/5/2015.
  */
-public class ComputeServiceImpl extends JcloudsComputeService implements ComputeService{
-    private static final String NOT_IMPLEMENTED_ERROR_MESSAGE = "Not implemented. Use 'amazon\' or 'openstack' providers in the provider input";
-
+public class ComputeServiceImpl extends JCloudsComputeService implements ComputeService {
```

Comment:
maybe you could rename all the service implementations using the "Impl" suffix like you did here? WDYT?